test/lit/string-builtins.wast

```diff
@@ -0,0 +1,60 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+
+;; Run normal -O2, which should lift, optimize, and lower strings, if the
+;; string builtins feature is enabled. We optimize in the last two here.
+
+;; RUN: foreach %s %t wasm-opt -O2 --enable-reference-types -S -o - | filecheck %s --check-prefix=MVP
+;; RUN: foreach %s %t wasm-opt -O2 -all -S -o - | filecheck %s --check-prefix=ALL
+;; RUN: foreach %s %t wasm-opt -O2 --enable-reference-types --enable-string-builtins -S -o - | filecheck %s --check-prefix=ESB
```

Comment:
If you enable GC here, the printed output should be the same as with `-all`, so you could use one fewer check prefix.

Reply:
Good idea, done.